{
	"header": {
		"line": "CVE-2019-19911 (There is a DoS vulnerability in Pillow before 6.2.2 caused by FpxImage ...)",
		"name": "CVE-2019-19911",
		"description": "(There is a DoS vulnerability in Pillow before 6.2.2 caused by FpxImage ...)"
	},
	"annotations": [
		{
			"line": "\t{DSA-4631-1 DLA-2057-1}",
			"type": "xref",
			"bugs": [
				"DSA-4631-1",
				"DLA-2057-1"
			]
		},
		{
			"line": "\t- pillow 7.0.0-1 (bug #948224)",
			"type": "package",
			"package": "pillow",
			"kind": "fixed",
			"version": "7.0.0-1",
			"flags": [
				{
					"bug": 948224
				}
			]
		},
		{
			"line": "\tNOTE: https://github.com/python-pillow/Pillow/commit/774e53bb132461d8d5ebefec1162e29ec0ebc63d (6.2.2)",
			"type": "NOTE",
			"description": "https://github.com/python-pillow/Pillow/commit/774e53bb132461d8d5ebefec1162e29ec0ebc63d (6.2.2)"
		}
	]
}
